Components/SplashArt.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { StyleSheet, Text, View, Image } from "react-native";
+import SplashImage from "../assets/bitmap.png";
+import { Platform } from "expo-core";
+export default class SplashArt extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.itemContainer}>
+          <Image source={SplashImage} />
+        </View>
+
+        <View style={styles.itemContainer}>
+          <Text style={styles.text}>share</Text>
+          <Text style={styles.text}>food with</Text>
+          <Text style={styles.text}>neighbors</Text>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: "row",
+    backgroundColor: "#fff"
+  },
+  itemContainer: {
+    justifyContent: "center"
+  },
+  imageContainer: {
+    width: "100%",
+    height: "100%"
+  },
+  text: {
+    fontSize: 35,
+    color: "#373504",
+    fontWeight: "bold",
+    alignSelf: "flex-end",
+    marginLeft: Platform.OS == "ios" ? 0 : -10
```

Comment:
Do these screen pictures appear?  In the future want me to pre-post screenshots if this occurs, or is there a way to avoid this behavior would be preferable.

Android no margin(rendered on my phone):
![screenshot_20190128-112805_expo](https://user-images.githubusercontent.com/38012334/51851088-28e3ef00-22f1-11e9-9e38-11f2a6f38d86.jpg)

Android margin -10:
![screenshot_20190128-112816_expo](https://user-images.githubusercontent.com/38012334/51851100-2ed9d000-22f1-11e9-9c36-47b6734f0af3.jpg)

Ios is fine as is simulator on iPhoneXR 12.1:
![screen shot 2019-01-28 at 11 33 09 am](https://user-images.githubusercontent.com/38012334/51851053-1bc70000-22f1-11e9-8cdc-24eb12743407.png)

Reply:
I'm pretty sure there might be a way to make the same effect happen while avoiding a negative margin. Could you try adding a padding to the container above the text?

Reply:
Talked with the group, going to first fit to smaller screen sizes, going with iPhone 6s, and Samsung galaxy  s7, currently no padding needed